<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoevolutionRunner</class>
 <widget class="QWidget" name="CoevolutionRunner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Sans Serif&quot;;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Coevolution Analysis</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="selectFileButton">
       <property name="text">
        <string>Select MSA File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="selectedFileLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QProgressBar" name="coevolutionProgress">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="runCoevolutionButton">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>91</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
